pricing/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Pricing</title>
+		<link rel="stylesheet" href="style.css" />
+	</head>
+	<body>
+		<header>
```

Comment:
Could you copy the Navbar from the profile page? I believe it's the most up-to-date version 
https://github.com/code-comp/code-comp.github.io/blob/main/profile/index.html#L16-L35